doc/changelog.rst

```diff
@@ -40,6 +40,12 @@ Version 0.5
 * Better navigation for long pages in the docs by adding a sidebar with links
   to subsections.
   (`PR 275 <https://github.com/fatiando/fatiando/pull/275>`__)
+* Added back-end support for decorators from duecredit
+  (`<https://github.com/duecredit/duecredit/>`__) to be added to methods. This
```

Comment:
Please replace

    duecredit  (`<https://github.com/duecredit/duecredit/>`__)` 

with 

    `duecredit  <https://github.com/duecredit/duecredit/>`__

The syntax

   `text <target>`__

in rst is equivalent to

    [Text](target)

In Markdown. I get this wrong all the time. That's one of the reasons i added "build the docs locally" to the check list :smile: